src/Algebra/Local/Structures.agda

```diff
@@ -0,0 +1,60 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Local algebraic structures
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Local.Structures where
```

Comment:
We should have a module `Algebra.Local` that re-exports both `Structures` and `Bundles`.

Reply:
I've put this in `Algebra.WithApartness`. One thing I wasn't sure of is whether this should import their definitions publicly. I assume that's the point?

Reply:
Yup, it should allow people to import all the definitions in the hierarchy with only one import statement.

Reply:
Great. As suggested on Zulip and noted below, everything is now at Algebra.Apartness.